package.json

```diff
@@ -36,7 +36,10 @@
     "storybook:build": "cross-env NODE_ENV=production NODE_NO_WARNINGS=1 storybook build"
   },
   "dependencies": {
+    "@emotion/react": "^11.11.0",
+    "@emotion/styled": "^11.11.0",
```

Comment:
These are `peer-dependencies` that `MUI` depends on

Reply:
Don't we use emotion already? 🤔